### ISSUE TITLE
HDR playback is not supported.

### ISSUE THREAD

**Issue by Weather-OS** [celluloid-player/celluloid#964]
Overview Description:
Whilst this project uses the MPV project as it's video player backend, it doesn't support HDR playback.

Steps to Reproduce:
1. Add the options `--vo=gpu-next --target-colorspace-hint=yes --gpu-api=vulkan --gpu-context=waylandvk` as you normally would in MPV inside "Extra MPV Options".
2. Play an HDR video.

Actual Results:
HDR doesn't get activated.

Expected Results:
HDR should get detected and activated.

Version: 0.27-1

**Comment by jannuary** [comment]
I don't think GTK supports HDR at the moment, and it would have to be landed there before we'll be able to implement it.

**Comment by Weather-OS** [comment]
Looks like it's shipping in 4.16. Any plans on implementing it?
https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/7444
https://docs.gtk.org/gdk4/struct.ColorState.html
https://blog.gtk.org/2024/08/11/the-colors-of-gtk/

**Comment by gnome-mpv** [comment]
> Looks like it's shipping in 4.16. Any plans on implementing it?

Sure, but I would wait until it's actually released.

**Comment by Weather-OS** [comment]
It's out.
https://gitlab.gnome.org/GNOME/gtk/-/tags/4.16.0

**Comment by MaxBrandtner** [comment]
So what's the status of this issue now that there seems to be nothing blocking hdr on celluloid?

**Comment by gnome-mpv** [comment]
It still needs to be implemented.

**Comment by jonkoops** [comment]
Looks like MPV just landed native support for HDR as well: https://github.com/mpv-player/mpv/releases/tag/v0.40.0

**Comment by Weather-OS** [comment]
> any updates on this? my pc support HDR yet no browser nor celluloid support HDR, I can't take advantage of this gorgeous AMOLED Panel :'(

Firefox supports HDR if you enable `gfx.wayland.hdr` and `layout.css.video-dynamic-range.allows-high`

**Comment by 5wHN28Dg** [comment]
> > any updates on this? my pc support HDR yet no browser nor celluloid support HDR, I can't take advantage of this gorgeous AMOLED Panel :'(
> 
> Firefox supports HDR if you enable `gfx.wayland.hdr` and `layout.css.video-dynamic-range.allows-high`

just tried it, yeah i can see it is still WIP but still thanks!

**Comment by Aliorpse** [comment]
OK so we don't have HDR even till now

**Comment by 5wHN28Dg** [comment]
> OK so we don't have HDR even till now

yeah... now I just use mpv whenever it is HDR video and https://mpv.rocks/manager/ made it very easy to use and setup mpv.
